Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:centos7
```

Comment:
please add building of this to `.travis.yml` (to build `MAVEN_CHECKS=true`)

Reply:
Do you mean to add this to the Maven build, or as a step after it has completed?

Reply:
after it has completed

Reply:
However, another option would be to add it to `presto-server/pom.xml`.``

Reply:
Currently added to `.travis.yml`, I'm concerned about adding it to `presto-server/pom.xml` as it could cause a build error if someone did not have Docker installed.

Reply:
Anyway, it should be under some maven profile. Not everyone would be interested in building docker image every time they build Presto.

Reply:
@kokosing I could make this part of the Maven build. It's definitely a bit more work, but I can make it happen if that would be preferred. I'm not sure what is going to fit better into the Presto build process.

Reply:
I've chatted with @electrum on Slack and his preference is to keep it separate from the Maven build at this time.